netbox/circuits/apps.py

```diff
@@ -6,4 +6,9 @@ class CircuitsConfig(AppConfig):
     verbose_name = "Circuits"
 
     def ready(self):
+        from netbox.models.features import register_model
```

Comment:
For DRY can we create a base NetBoxAppClass and have a base ready that registers models?  Or in lieu of that have a register_all_models(instance) that does the loop / register_model.